agent/hibinit-agent

```diff
@@ -89,25 +89,25 @@ def fallocate(fl, size):
         if res != 0:
             raise Exception("Failed to perform fallocate(). Result: %d" % res)
     except Exception as e:
-        log("Failed to call fallocate(), will use resize. Err: %s" % str(e))
-        fl.seek(size-1)
+        print_to_sys_log("Failed to call fallocate(), will use resize. Err: %s" % str(e))
+        fl.seek(size - 1)
         fl.write(chr(0))
 
 
 def get_file_block_number(filename):
     with open(filename, 'r') as handle:
         buf = array.array('L', [0])
         # from linux/fs.h
-        FIBMAP = 0x01
-        result = fcntl.ioctl(handle.fileno(), FIBMAP, buf)
+        fibmap = 0x01
```

Comment:
should we use it as a constant globally ?

Reply:
I suppose this is possible but I don't see the real benefit, as it will not be used in any other part of the code.